CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* [refactor] Remove obsolete scripts.
```

Comment:
I guess that could also qualify as "chore", but it doesn't make much of a difference. :)

Reply:
Yes, I was going back and forth between those but finally decided that it is a refactor since #13 was a refactor and this is a continuation of that.

Reply:
Fair!